### PR TITLE
Switch over to workflow releases from messy artifact downloads 

### LINKED ISSUE #217: download link in readme not working
also wouldnt it be easier to release on github / use actions?

### PR DESCRIPTION
**Pull request by CyberFlameGO**
Closes #217

### CODE REVIEW

File: .github/workflows/build-nachospigot.yml [hunk 1]
```diff
@@ -1,32 +1,51 @@
...
+    - uses: actions/upload-artifact@v2
+      if: ${{ matrix.java == 8 }}
       with:
-        name: NachoSpigot-server
-        path: ${{ github.workspace }}/NachoSpigot-Server/target/NachoSpigot.jar
-
-    - name: Upload API Artifact
-      uses: actions/upload-artifact@v2.2.0
+        name: NachoSpigot
+        path: ./NachoSpigot-Server/target/NachoSpigotSpigot.jar
```
Comment (Elierrr) [inline .github/workflows/build-nachospigot.yml:28]:
```suggestion
        path: ./NachoSpigot-Server/target/NachoSpigot.jar
```

**Review by Elierrr — CHANGES_REQUESTED** [review]
I don't agree with this, but here's a small fix if it does get merged.

**Comment by CyberFlameGO** [comment]
> I don't agree with this

how come?

**Comment by Elierrr** [comment]
> how come?

It'll spam the releases. We should probably have it modify an existing release.

**Comment by CyberFlameGO** [comment]
I agree, but I don't know of a better solution apart from a Jenkins CI, since this nightly.link thing isn't a great solution

**Comment by Elierrr** [comment]
> 
> 
> I agree, but I don't know of a better solution apart from a Jenkins CI, since this nightly.link thing isn't a great solution

It works fine on any other repository, including repositories that don't have it authorized. Not sure why it stopped working now.

**Comment by Lucaskyy** [comment]
It stopped working since someone changed the artifact name without me noticing, but it's all fixed now.

**Pull request closed without merging**